pkgs/development/tools/analysis/ida-free/default.nix

```diff
@@ -0,0 +1,113 @@
+{ lib
+, stdenv
+, fetchurl
+, libGL
+, zlib
+, libsecret
+, libxkbcommon
+, glib
+, makeWrapper
+, openssl
+, xorg
+, dbus
+, fontconfig
+, freetype
+, makeDesktopItem
+}:
+stdenv.mkDerivation rec {
+  pname = "ida-free";
+  version = "8.2.230124";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20230213091148if_/https://out7.hex-rays.com/files/idafree82_linux.run";
+    sha256 = "sha256-pcyb8M39FJVsWUAQiXy17nOtVUhPkPsE24YqhK9rK58=";
+  };
+
+  icon = fetchurl {
+    urls = [
+      "https://www.hex-rays.com/products/ida/news/8_1/images/icon_free.png"
+      "https://web.archive.org/web/20221105181231if_/https://hex-rays.com/products/ida/news/8_1/images/icon_free.png"
+    ];
+    sha256 = "sha256-widkv2VGh+eOauUK/6Sz/e2auCNFAsc8n9z0fdrSnW0=";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = "ida64";
+    icon = icon;
+    comment = "Freeware version of the world's smartest and most feature-full disassembler";
+    desktopName = "IDA Free";
+    genericName = "Interactive Disassembler";
+    categories = [ "Development" ];
+  };
+
+  ldLibraryPath = lib.makeLibraryPath [
```

Comment:
debugger:

> libunwind: failed to load libunwind-x86_64.so.8

Reply:
cannot reproduce, cannot find `libunwind` in symbols. Is it in IDA or is it the program being debugged?

Reply:
its missing in ida

```
$ grep -r libunwind-x86_64.so.8 result/
grep: result/opt/ida-free/ida.hlp: binary file matches
grep: result/opt/ida-free/plugins/linux_user64.so: binary file matches
```